CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 Diff for [unreleased]
 
+## 2.2.5 - 2018-02-23
```

Comment:
Could be 2018-02-26

Reply:
Totally Forgot to change it. I just pushed a fix.